composer.json

```diff
@@ -82,5 +83,7 @@
         "link-templates": [
             "ln -s $(pwd)/src/Resources/views/bundles/* $(pwd)/tests/Application/templates/bundles"
         ]
-    }
+    },
+    "minimum-stability": "dev",
```

Comment:
Can we instead specify `sylius/sylius: ~1.10.0 || ~1.11.0 || 1.12-dev` or something similar that would work? we need dev dependency only for Sylius 🖖